superset-frontend/src/SqlLab/components/App/index.jsx

```diff
@@ -96,29 +94,14 @@ class App extends React.PureComponent {
   }
 
   render() {
-    let content;
     if (this.state.hash && this.state.hash === '#search') {
-      if (isFeatureEnabled(FeatureFlag.ENABLE_REACT_CRUD_VIEWS)) {
```

Comment:
I kept this logic the same as before, but the logic doesn't look quite right to me. Shouldn't we be rendering with react instead of `location.replace`ing when react crud is enabled?